release/ray_release/configs/global_config.py

```diff
@@ -4,6 +4,8 @@
 
 class GlobalConfig(TypedDict):
     byod_ray_ecr: str
+    byod_ray_cpu_project: str
+    byod_ray_gpu_project: str
```

Comment:
this should be named as `ray_ml` not `ray_gpu`.

`rayproject/ray` also has `gpu` tag. `gpu` tag means that the container is based on some version of `cuda` image. `ml` tag means that a big list of additional apt and python packages are installed in the image.